std/datetime.d

```diff
@@ -611,6 +611,14 @@ struct SysTime
 public:
 
     /++
+			  Creates a SysTime from a DateTime and TimeZone. The DateTime is taken to represent the
+				current time as one would see on a clock in that timezone.
+
+				In the case of daylight saving time, some times are ambiguous; for instance, in the
+				America/Los_Angeles timezone, 2015-11-01T01:00:00 occurred twice. If the given DateTime
+				represents an ambiguous time of this sort, it will be considered as the earliest hour with
+				the relevant number.
```

Comment:
The additions to this ddoc comment make sense, but IMHO, the rest of the PR does not.